composer.json

```diff
@@ -37,7 +37,13 @@
         "vimeo/psalm": "4.7.1"
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
```

Comment:
What's this?

Reply:
>As of Composer 2.2.0, the allow-plugins option adds a layer of security allowing you to restrict which Composer plugins are able to execute code during a Composer run.

It's added by composer when you run `require` or `update` on version `2.2` and above.

ref: https://getcomposer.org/doc/06-config.md#allow-plugins